views/templates/front/payment-method.tpl

```diff
@@ -27,26 +27,53 @@
     </form>
 {else}
     <div class="row adyen-payment {$paymentMethodType|escape:'html':'UTF-8'}"
-         data-local-payment-method="{$paymentMethodType|escape:'html':'UTF-8'}"
-    >
+         data-local-payment-method="{$paymentMethodType|escape:'html':'UTF-8'}">
+        {if $isPrestaShop16}
+        <div class="col-xs-12 col-md-12">
+            <div class="payment_module adyen-collapser collapsed" data-toggle="collapse"
+                 data-target="#collapse{$paymentMethodType|escape:'html':'UTF-8'}" aria-expanded="false"
+                 aria-controls="collapse{$paymentMethodType|escape:'html':'UTF-8'}"
+                 style="background-image: url(https://checkoutshopper-live.adyen.com/checkoutshopper/images/logos/medium/{$paymentMethodBrand|escape:'html':'UTF-8'}.png)">
```

Comment:
could you please create a ticket to replace the burnt in image links to resources in the prestashop environment? Similar to the ticket: PW-3120